src/components/views/settings/encryption/ResetIdentityBody.tsx

```diff
@@ -36,22 +37,24 @@ interface ResetIdentityBodyProps {
 }
 
 /**
- * "compromised" is shown when the user chooses 'reset' explicitly in settings, usually because they believe their
- * identity has been compromised.
+ * "compromised" is shown when the user chose 'Reset cryptographic identity' explicitly in settings, usually because
+ * they believe their identity has been compromised.
  *
  * "sync_failed" is shown when the user tried to recover their identity but the process failed, probably because
  * the required information is missing from recovery.
  *
- * "forgot" is shown when the user has just forgotten their passphrase.
+ * "forgot" is shown when the user chose 'Forgot recovery key?' during `SetupEncryptionToast`.
+ *
+ * "confirm" is shown when the user chose 'Reset all' during `SetupEncryptionBody`.
```

Comment:
it would be good to detail how the behaviour is different, rather than when it is called.

Reply:
No behaviour change.  It just changes the messages displayed.  I've added a comment about that.